adapters/lock_win32.c

```diff
@@ -31,7 +31,7 @@ LOCK_RESULT Lock_Deinit(LOCK_HANDLE handle)
     LOCK_RESULT result;
     if (handle == NULL)
     {
-        /* Codes_SRS_LOCK_10_007: [Lock_Deinit on NULL handle passed returns LOCK_ERROR] */
+        /* Codes_SRS_LOCK_10_013: [ `Lock_Deinit` on NULL `handle` passed returns `LOCK_ERROR` ]*/
```

Comment:
>[ ` [](start = 34, length = 3)

nit: backticks

Reply:
Fixed.

---
In reply to: [362558185](https://github.com/Azure/azure-c-util/pull/7#discussion_r362558185) [](ancestors = 362558185)